packages/dev-tools/tsconfig.json

```diff
@@ -9,7 +9,9 @@
     "isolatedModules": true,
     "esModuleInterop": true,
     "jsx": "react-jsx",
-    "jsxImportSource": "react"
+    "jsxImportSource": "react",
+    "exactOptionalPropertyTypes": true,
+    "skipLibCheck": true
```

Comment:
I don't love enabling this here but otherwise, `exactOptionalPropertyTypes: true` causes a bunch of random react-router issues that we can't actually fix.